src/testsuite/vec_perf_i128.c

```diff
@@ -500,13 +504,14 @@ timed_ctmaxdouble_10e32 (void)
   double d = 0;
   int rc = 0;
 
+#ifndef __clang__
```

Comment:
why? add a comment.

Reply:
Can't call strtod() on from clang

/* Disable for __clang__ because of bug involving <floatn.h>
   incombination with -mcpu=power9 -mfloat128 */